Fix panel width

TKTrader opened the pull request:
Fixed box width behind logins
Updated node packages
Added Olympics html icon

ssamkough committed:
Added login and changed some of the routing

Commit:
Revert "Added login and changed some of the routing"

This reverts commit 03f728abf09d93ec8de5b78e5cd0be77f15d5dfa.

Commit:
Merge branch 'DevDFS' of https://github.com/TKTrader/SOGs into DevDFS

Commit:
added html icon, fixed login backgrounds, updated node packages

dfsim commented:
It looks like we have some type of discrepancy on how its coming up. I tested it on my chrome, and the padding is a bit too much on the right. I sent you a few screen shots on discord.

dfsim committed:
Update login.component.scss

Changed max-width back, bottom padding looks good. This pull request and the other have the same change so I'll edit this one here and we can discuss any more changes on the other pull.

dfsim committed:
Update register.component.scss

Changed max-width back, bottom padding looks good. This pull request and the other have the same change so I'll edit this one here and we can discuss any more changes on the other pull.

dfsim commented:
Edited max-width back to 350 from 475 for login/register, this change is the same in the other pull request so we can see what works better there.
